kyuubi-server/src/test/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilderSuite.scala

```diff
@@ -225,6 +224,21 @@ class SparkProcessBuilderSuite extends KerberizedTestHelper {
     val b2 = new SparkProcessBuilder("test", conf)
     assert(b2.mainResource.getOrElse("") != jarPath.toString)
   }
+
+  test("test kill application") {
+    val config = KyuubiConf()
+    val pb: FakeSparkProcessBuilder = new FakeSparkProcessBuilder(config)
+    val lastRowOfLog = "Application report for application_1593587619692_19713 (state: ACCEPTED)"
+
+    val e = intercept[KyuubiSQLException] {
+      pb.killApplication(lastRowOfLog)
+    }
+
+    assert(e.getMessage === "Failed to kill yarn job. " +
```

Comment:
It's not a robust test. The test will fail if the user set `HADOOP_HOME` in their machine.